core/script_language.h

```diff
@@ -254,7 +254,8 @@ class ScriptLanguage {
 	virtual String debug_parse_stack_level_expression(int p_level, const String &p_expression, int p_max_subitems = -1, int p_max_depth = -1) = 0;
 
 	struct StackInfo {
-		Ref<Script> script;
+		String file;
+		String func;
 		int line;
```

Comment:
Forgot to mention I changed the StackInfo struct. It includes the file path and the function string (not necessarily the name, it can be formatted to include the signature). I also removed the script from it because it was not needing and it meant you couldn't include files outside the project.